### PR TITLE
Getting help and helping out

### PR DESCRIPTION
**Pull request by rolandlo**
This PR adds a section on where/how to get help and how to contribute to Xournal++. I listed all the places and described them briefly, trying to make clear what belongs where and what we expect.

### CODE REVIEW

File: docs/guide/help.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Getting help and helping out
+
+## Website documentation and user manual
+
+This website is intended to describe Xournal++ and its features reasonably well and complete, so that the average user can find out how Xournal++ works. We are working on
+completing its content. The website will replace the (rather incomplete) [User manual](https://github.com/xournalpp/xournalpp/wiki/User-Manual) on the Github repository.
+More technical questions (like how to build Xournal++ from source, or how to check out a pull request) that are relvant for technically more advanced users will still be
```
Comment (Technius) [inline docs/guide/help.md:7]:
```suggestion
More technical questions (like how to build Xournal++ from source, or how to check out a pull request) that are relevant for technically more advanced users will still be
```

File: docs/guide/help.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Finding out what other people are using Xournal++ for
+- Getting help with installation or setting up a feature
+- etc.
+
+We hope to grow our community knowledge base here.
+
+## Github issue tracker
+
+Use the [issue tracker](https://github.com/xournalpp/xournalpp/issues) on the Github repository to report bugs and propose enhancements/new features.
+Please check beforehand if the same bug has already been reported or the same enhancement has already been suggested. If so, use the existing ticket to express yourself.
```
Comment (Technius) [inline docs/guide/help.md:26]:
```suggestion
Please check beforehand if the same bug has already been reported or the same enhancement has already been suggested. If so, use the existing ticket to add additional information or suggest improvements.
```

File: docs/guide/help.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+This will make it easier for us to keep everything under control. Duplicates will usually be closed rapidly, but it always takes time to find them, which the develeopers
+would prefer to invest in coding and improving the documentation. Issues that are resolved will be closed as well. Open questions are better discussed on Github discussions.
+
+## Gitter channel
```
Comment (Technius) [inline docs/guide/help.md:30]: I would also suggest adding a link to the Matrix / Element channel.

File: docs/guide/help.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+If you want to discuss some topic related to Xournal++ in chat mode, you may find someone to talk with in the [Gitter channel](https://gitter.im/xournalpp/xournalpp).
+
+## The Crowdin project
+
+Our translations are managed in the [Crowdin project](https://crowdin.com/project/xournalpp). If you would like to contribute, create an account and start contributing.
+While some languages like English, German and Italian are basically complete, other languages require more care. If you are interested in translating a new language,
+contact us on Gitter or create a new issue and we will unlock the language on Crowdin.
```
Comment (Technius) [inline docs/guide/help.md:38]:
```suggestion
Our translations are managed in the [Crowdin project](https://crowdin.com/project/xournalpp). If you would like to contribute, create an account and start contributing.
While some languages like English, German and Italian are basically complete, most other languages still need more work. If you are interested in translating a new language,
ask the project maintainers on Gitter or create a new issue and they will unlock the language on Crowdin.
```

File: docs/guide/help.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Github pull requests
+
+If you want to contribute to the codebase of Xournal++ you may create a [pull request on Github](https://github.com/xournalpp/xournalpp/pulls). Once the pull request is
+ready for review, one of the developer team will review it, suggest changes if necessary or accept/decline the pull request. If you plan some feature, that has not been
+requested in the issue tracker yet, better open an issue there first, so you know whether your work will be appreciated. Pull requests can be linked to specific issues,
+that will get automatically closed when the pull request is merged.
```
Comment (Technius) [inline docs/guide/help.md:45]: Last sentence here isn't really informative for contributors, so I would just remove it.

**Review by Technius — COMMENTED** [review]
I think this would be better placed on its own top-level section under the title "Community", as most of these resources are really about interacting with the Xournal++ community. "Guide", as I understand it, is better left as a manual on how to use the software.

**Comment by rolandlo** [comment]
@Technius Thanks for your review. I have made all the changes you suggested. However if I try it out locally (via `mkdocs serve`) I get some mixed up navigation bars. Can't figure out what is wrong.
grafik

**Pull request merged**